### PR TITLE
qa snackbar, add action buttons

### PR DESCRIPTION
**Pull request by IvarBorthen**
• Redesign
• Label (close) from i18n
• Can add button (generic for stuff like show/regret etc)

### CODE REVIEW

File: packages/designmanual/stories/molecules/SnackbarExample.tsx [hunk 1]
```diff
@@ -46,10 +46,18 @@ const SnackBarExample = () => {
...
+            ariaLabel: 'Console.log onClick event',
+            onClick: () => {
+              console.log('Vis'); // eslint-disable-line no-console
+            },
+          },
+        ]}
         onKill={(id: string | undefined) => {
           // eslint-disable-next-line
-          console.log(`snack with id ${id || 'unknown'} removed`);
+          console.log(`snack with id ${id || 'unknown'} removed`); // eslint-disable-line no-console
```
Comment (peterrolfsen) [inline packages/designmanual/stories/molecules/SnackbarExample.tsx:60]: Skal console.log være med?

File: packages/designmanual/stories/molecules/SnackbarExample.tsx [hunk 2]
```diff
@@ -46,10 +46,18 @@ const SnackBarExample = () => {
         key={snack.snackbarItemId}
         type={snack.type}
         snackbarItemId={snack.snackbarItemId}
-        closeAriaLabel="Lukk"
+        actionButtons={[
+          {
+            text: 'Vis',
+            ariaLabel: 'Console.log onClick event',
+            onClick: () => {
+              console.log('Vis'); // eslint-disable-line no-console
```
Comment (peterrolfsen) [inline packages/designmanual/stories/molecules/SnackbarExample.tsx:54]: Skal console.log være med?

File: packages/designmanual/stories/molecules/SnackbarExample.tsx (hunk 1 above)
Comment (IvarBorthen) [reply packages/designmanual/stories/molecules/SnackbarExample.tsx:60]: Er bare i storybook'en..

File: packages/designmanual/stories/molecules/SnackbarExample.tsx (hunk 2 above)
Comment (IvarBorthen) [reply packages/designmanual/stories/molecules/SnackbarExample.tsx:54]: same as above

File: packages/designmanual/stories/molecules/SnackbarExample.tsx (hunk 1 above)
Comment (peterrolfsen) [reply packages/designmanual/stories/molecules/SnackbarExample.tsx:60]: ahh ja sett

**Pull request merged**